harvard-lts/spotlight-oaipmh-resources
# Updates

This is a security update set of changes, as well as a fix for solr harvesting due to an update in Ruby 3.2 syntax that broke it

To test:
- point your local Gemfile at this branch, and rebuild. Make sure your Gemfile.lock shows:
  GIT
  remote: https://github.com/harvard-lts/spotlight-oaipmh-resources
  revision: 936c11fd9b7460eeed68cfdb9bb0599e6bb66819
  branch: updates
  specs:
    spotlight-oaipmh-resources (3.0.2)
      mods
      oai
      
- run a solr harvest, I suggest acats-calendar or tedming as they aren't gigantic. You should not have an immediate failure (what is happening now on qa/prod).
- run a oai harvest (demo), should work
